src/dev/build/build_distributables.ts

```diff
@@ -64,10 +64,11 @@ export async function buildDistributables(log: ToolingLog, options: BuildOptions
   if (options.createGenericFolders) {
     await run(Tasks.CopySource);
     await run(Tasks.CopyBinScripts);
-    await run(Tasks.ReplaceFavicon);
+
     await run(Tasks.CreateEmptyDirsAndFiles);
     await run(Tasks.CreateReadme);
     await run(Tasks.BuildBazelPackages);
+    await run(Tasks.ReplaceFavicon);
```

Comment:
The build process expects an empty directory into which to replace the distributable versions of favicons with their source. At least, that's how I understand the flow here.